torch/csrc/profiler/data_flow.cpp

```diff
@@ -33,62 +68,37 @@ void calculateUniqueTensorIDs(
   // Flatten results to a uniform representation.
   // --------------------------------------------------------------------------
   {
-    auto insert_tensor = [&tensors](TensorMetadata& m) {
-      tensors.emplace_back(RawTensorInfo{
-          m.impl(),
-          m.data_,
-          m.device(),
-          /*is_free_=*/false,
-          m.allocation_id_,
-          m.id_});
-    };
-
+    RawTensors raw_tensors;
     for (auto& result : sorted_results) {
       result->visit(c10::overloaded(
           [&](ExtraFields<EventType::TorchOp>& torch_op) {
-            for (auto& m : torch_op.inputs_.tensor_metadata_) {
-              if (m.has_value()) {
-                insert_tensor(*m);
-              }
+            for (auto& i : torch_op.inputs_) {
+              c10::visit(raw_tensors, i);
             }
           },
-          [&](ExtraFields<EventType::Allocation>& alloc_op) {
```

Comment:
we are not catching tensors from allocation calls any more?

Reply:
We still are. `[&](auto& i) { raw_tensors(i); }));` forwards to `RawTensors::operator()(ExtraFields<EventType::Allocation>&)`